tensorflow/python/keras/benchmarks/keras_examples_benchmarks/bidirectional_lstm_benchmark_test.py

```diff
@@ -21,14 +23,14 @@
 
 import tensorflow as tf
 
-from tensorflow.python.keras.benchmarks import benchmark_util
 from tensorflow.python.platform import benchmark
+from tensorflow.python.keras.benchmarks import benchmark_util
 
 _MAX_FEATURE = 20000
 _MAX_LEN = 200
```

Comment:
This can be moved to the constructor as well, as global variables are not necessary any more.

Reply:
Sounds great.